streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -189,8 +189,9 @@ public void onPartitionsAssigned(final Collection<TopicPartition> assignment) {
                 log.trace("{} pausing partitions: {}", logPrefix, partitions);
                 consumer.pause(partitions);
             } catch (final Throwable t) {
+                log.error("{} Error caught during partition assignment, " +
```

Comment:
nit: put the error message earlier in the log statement  `Error caught during partition assignment {},`